test/pdfs/.gitignore

```diff
@@ -70,6 +70,7 @@
 !bug1650302_reduced.pdf
 !issue8570.pdf
 !issue8697.pdf
+!issue12909.pdf
```

Comment:
Shouldn't this be a linked test-case instead?

Reply:
I don't know.
How do you decide this in general ?

Reply:
I don't think we have a rule, but generally I follow:
- reduced test case - include in repo
- small file author has given us permission to use - include in repo
- linked that will likely keep working (e.g. github, bugzilla) - linked test case
- linked file from random site - try to create reduced test case or use way back machine